src/ex4/server/services/item_manager.js

```
@@ -0,0 +1,89 @@
+const PokemonClient = require("../clients/pokemon_client");
+const { Item } = require("../db/models/item");
+
+class ItemManager {
+  constructor() {
+    this.pokemonClient = new PokemonClient();
+  }
+
+  getItems = () => {
+    try {
+      return Item.findAll();
+    } catch (err) {
+      console.error(err);
```

suppressing the error doesn't really helps us in case of an exception. it's great that you're logging the failure, but we need to handle it as well. right now all you do is returning an undefined object.

there are many ways to handle this scenarios, but most importantly is to think "how the client will know there was an issue?"